src/Backups/IBackup.h

```diff
@@ -85,7 +91,7 @@ class IBackup : public std::enable_shared_from_this<IBackup>
     virtual bool supportsWritingInMultipleThreads() const = 0;
 };
 
-using BackupPtr = std::shared_ptr<const IBackup>;
+using BackupPtr = std::shared_ptr<IBackup>;
```

Comment:
Why? We have `BackupMutablePtr` for this. The idea of `BackupPtr` was to contains a "const backup".

Reply:
My fault. Fixed it